src/app.js

```diff
@@ -155,7 +149,7 @@ export default function (app) {
 
       for (var name in node.data) {
         if (name === "onCreate") {
-          defer(node.data[name], element)
+          node.data[name](element)
```

Comment:
calling the onCreate handler before we've created the child elements means that the element is empty when the oncreate handler is called.  It decreases the utility of the lifecycle method. I suggest moving the eventual call to onCreate handlers until after the child-node creation loop.

As a motivation to why I think it matters:

I noticed this because one of my tests depends on the children being available to the element (so I can loop over them). If left as is, I would have to wrap my onCreate in a setTimeout function which feels funky, and like something I shouldn't have to do.